pd/src/staking.rs

```diff
@@ -1,12 +1,32 @@
+use num::bigint;
+use num::rational;
+use penumbra_crypto::{Address, Value};
+use serde::{Deserialize, Serialize};
 use tendermint::{vote, PublicKey};
 
 const PENUMBRA_BECH32_VALIDATOR_PREFIX: &str = "penumbravalpub";
 /// Validator tracks the Penumbra validator's long-term consensus key (tm_pubkey), as well as their
 /// voting power.
-#[derive(Debug, Eq, PartialOrd, Ord, Clone, Copy)]
+#[derive(Deserialize, Serialize, Debug, Eq, Clone)]
```

Comment:
What are we using this with Serde for? To load it from the `genesis.json` file, right?

Reply:
Yes, we're loading `Validator`s from the application genesis state in JSON format.